tests/test_templates_embeddings.py

```diff
@@ -148,6 +148,26 @@ def circuit(x=None):
         with pytest.raises(ValueError, match="'features' must be of shape"):
             circuit(x=inpt)
 
+    def test_amplitude_embedding_tolerance_value(self):
+        """Tests that a small enough tolerance value is used for Amplitude
+        Embedding."""
+        inputs = np.array([0.2590635 , 0.11604707, 0.17591636, 0.16552171, 0.01634442,
+                           0.00661965, 0.22336007, 0.16188534, 0.23487748, 0.08265884,
+                           0.29210808, 0.29547198, 0.06001319, 0.05693569, 0.274379  ,
+                           0.16366717, 0.04848188, 0.29243237, 0.29257721, 0.00785323,
+                           0.21839711, 0.32690427, 0.16370515, 0.27559101, 0.10555633])
+
+        num_qubits = 5
+        dev = qml.device('default.qubit', wires=num_qubits)
+        assert np.isclose(np.sum(np.abs(inputs) ** 2), 1, 10e-4)
```

Comment:
How comes you use 10e-4? 

Wouldn't the test fail to catch issues between the qnode tolerance value of 1e-10 and 10e-4?

Reply:
Right! So this was coming from the fact that for an input array that was within `10e-9` tolerance of `1` using `np.isclose` returned `True` not only for `10e-9`, but also for `10e-10`. There are more on this in the #534 issue on this as a whole, I'll update this PR once we've come to a decision there. :smile: